docs/framework/guides/contributing/code-style.md

```diff
@@ -604,6 +604,56 @@ this.canObserve;
 this.mustRefresh;
 ```
 
+### Buttons, Commands and Plugins
```

Comment:
As I can see, we're putting a blank line before the list markup.

```md
### Header

A paragraph:

* List item
* List item 2
```

I'll fix it.